fgcm/fgcmDeltaAper.py

```diff
@@ -450,7 +450,7 @@ def computeEpsilonPerCcd(self):
         # ignore bands that have a range > 10
         deltaRange = scaleRange[:, 1] - scaleRange[:, 0]
         filtersToMatchRange, = np.where(deltaRange < 20.0)
-        if filtersToMatchRange.size == 0:
+        if filtersToMatchRange.size > 0:
```

Comment:
Squash this into https://github.com/lsst/fgcm/pull/42/commits/d8c6a71cc4dda8977860026a3f78562b2c2715dc (or is it too late?)